src/mmrelay/plugins/ping_plugin.py

```
@@ -38,7 +38,21 @@ def description(self):
 
     async def handle_meshtastic_message(
         self, packet, formatted_message, longname, meshnet_name
```

Ruff (0.14.6)

Unused method argument: `formatted_message`

(ARG002)